### DISCUSSION TITLE
pp calculation of lazer scores is not completing on website

### DISCUSSION THREAD

**Discussion by sirlupusdev** [ppy/osu#27805] [Q&A] [answered]
Hello,

I recently started setting some scores using osu!lazer. However, all new scores set on lazer still show as calculating on the website, no matter whether lazer mode is enabled or not. All scores are set with the Classic mod enabled.
This includes these scores for example: 
[https://osu.ppy.sh/scores/2624739818](https://osu.ppy.sh/scores/2624739818) (about 2 days ago as of writing)
[https://osu.ppy.sh/scores/2630106348](https://osu.ppy.sh/scores/2630106348) (about 1 day ago as of writing)

The calculation is completed (pp is shown) when checking the details screen in game which can also be seen here: https://drive.google.com/file/d/1fPa84LgY9I8V0pv02hqoPkLefliU1APZ/view
Scores set on stable still work fine (as of a few days ago).

This discussion was created after a brief conversation with osu! support.

Thanks for your help in advance.

**Comment by Walavouchey** [comment]
@ppy/team-client is it expected for pp to take days to calculate?

**Reply by bdach** [reply, marked as answer]
these are scores on lazer with classic mod enabled. ergo, they don't give pp.

this is probably a display-only regression on the website. i mentioned this already [elsewhere](https://github.com/ppy/osu-queue-score-statistics/pull/227#issuecomment-1969827044). @ppy/team-web

**Reply by sirlupusdev** [reply]
Oh, alright, thank you. Is there any specific reason for this (as it is ranked on stable)?

**Reply by bdach** [reply]
it's because of weird stable mechanics and the complications they cause (lazer classic mod doesn't completely match the stupid handling of combo for sliders).

**Reply by sirlupusdev** [reply]
Thank you. Looks like I need to look into getting my slider accuracy to work or go back to stable if I want to earn pp.

**Reply by bdach** [reply]
mind you that doesn't mean that we won't eventually rank classic (in fact we probably want to), but no estimates on this as of this moment.
